net/socks5/response.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package socks5
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+)
+
+const maxResponsePacketSize = 261
+
+// Reply are the bytes sent in SOCKS5 packets
+// that represent replies from the server to a client
+// request.
+type Reply byte
```

Comment:
this probably doesn't need to be exported anymore, if this isn't going to be a socks library for external use.

and then maybe call it `replyCode` instead?